src/trinary_sse42.h

```diff
@@ -9,8 +9,16 @@
 #ifndef TRINARY_SSE42_H_
 #define TRINARY_SSE42_H_
 
+#if defined(__SSE4_2__)
 #include <nmmintrin.h>
+#endif
```

Comment:
Can you move `sse2neon.h` right after `nmmintrin.h`?

Reply:
I would like to.
But I was thinking about the alphabetical order of the header files.

Reply:
Since `sse2neon.h` is not a system header file, I keep it in the same place.